.github/workflows/deploy-styleguide.yml

```diff
@@ -23,11 +23,10 @@ jobs:
       - name: Build & deploy
         shell: bash -l {0}
         run: |
-          git config user.name github-actions
-          git config user.email github-actions@github.com
```

Comment:
Since you removed this, I think you gotta set the cwd on these commands to the dist directory:

https://github.com/square/maker/blob/master/build/deploys/push.js#L15

Reply:
That's a good point.... how in the world did it work before though 🤔

Reply:
Ohhh, it worked before because both `pull.js` and `push.js` call `ensure.js` which changes the directory to the deploy directory here: https://github.com/square/maker/blob/master/build/deploys/ensure.js#L13

Anyway, that's a bit too easy to miss, so I made it more explicit in this commit: https://github.com/square/maker/pull/70/commits/fcde22398068ce924af4ed20a7b3806a5be3c50b